package.json

```diff
@@ -5,16 +5,18 @@
   "main": "ping-centre.js",
```

Comment:
This also looks fishy. I don't think we have a ./ping-centre.js anymore. We'll either need to redirect this to:

- src/ping-centre.js
- dist/ping-centre.min.js
- \<other\>